scripts/tests/test262/test262_whitelist.txt

```diff
@@ -10,102 +10,22 @@ language/expressions/async-arrow-function/early-errors-arrow-await-in-formals-de
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-heritage-call-expression-bad-reference.js(strict mode)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-heritage-call-expression-this.js(default)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-heritage-call-expression-this.js(strict mode)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-heritage-member-expression-bad-reference.js(default)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-heritage-member-expression-bad-reference.js(strict mode)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-heritage-member-expression-this.js(default)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-heritage-member-expression-this.js(strict mode)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-member-expression-bad-reference.js(default)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-member-expression-bad-reference.js(strict mode)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-member-expression-this.js(default)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-member-expression-this.js(strict mode)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-outter-call-expression-bad-reference.js(default)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-outter-call-expression-bad-reference.js(strict mode)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-outter-call-expression-this.js(default)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-outter-call-expression-this.js(strict mode)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-outter-member-expression-bad-reference.js(default)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-outter-member-expression-bad-reference.js(strict mode)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-outter-member-expression-this.js(default)
-language/expressions/class/elements/syntax/early-errors/invalid-names/method-outter-member-expression-this.js(strict mode)
-language/expressions/class/elements/syntax/early-errors/super-private-access-invalid.js(default)
-language/expressions/class/elements/syntax/early-errors/super-private-access-invalid.js(strict mode)
 language/expressions/object/method-definition/private-name-early-error-async-gen-inside-class.js(default)
 language/expressions/object/method-definition/private-name-early-error-async-gen-inside-class.js(strict mode)
+language/expressions/object/method-definition/private-name-early-error-async-gen.js(default)
```

Comment:
This test (correctly) threw but for the wrong reason. I will open a [good first issue] for it after that this PR is merged.